ratings/schema.sql

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+CREATE TABLE IF NOT EXISTS models (
+  id integer primary key,
+  model_name text,
+  bucket text,
+  model_num integer not null,
+
+  num_games integer not null,
+  num_wins integer not null,
+
+  ranking float,
+  var float,
+
+  black_games integer,
+  black_wins integer,
+  white_games integer,
+  white_wins integer,
+  unique(bucket, model_name)
+);
+CREATE INDEX IF NOT EXISTS model_name_bucket_index ON models (model_name, bucket);
```

Comment:
move this to the bottom with the other indexes

Reply:
Done.